LondonBritishLibrary/orient/BLorient793.xml

```diff
@@ -26,14 +25,781 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+     <textLang mainLang="gez"></textLang>
+     <incipit xml:lang="gez">ጸሎተ፡ ሃይማኖት፡ ዘአፍራቅያ፡ ይቤ፡ ኵሉ፡ ለለአሐዱ፡ ዘይፈቅድ፡ ይድኃን፡ ይኅሥሥ፡ ሃይማኖተ፡ ኮቶሊክ፡ እንተ፡ ይእቲ፡ ሐዋርያት፡ እንተ፡ ትሜህር፡ ጽምረተ፡ ሥላሴ፡
+      በኢቱሳሔ፡ ወድሙረ፡ በኢቡኣዴ፡ አብ፡ በአካሉ፡ ወበግጻዌሁ፡ ዘኢይትዌለጥ፡ እምከዊነ፡ አብ፡ ኀበ፡ ከዊነ፡ ወልድ፡ ወመንፈስ፡ ቅዱስ። </incipit>
+     </msItem>
+
+     <msItem xml:id="ms_i2.24">
+     <locus from="110r"/>
+     <title xml:lang="en" ref="LIT5441ApostolicCreed">The Apostles's Creed</title>
+     <textLang mainLang="gez"></textLang>
+     <incipit xml:lang="gez">ጸሎተ፡ ሃይማኖት። ንአምን፡ በ፩አምላክ፡ እግዚአብሔር፡ አብ፡ አኃዜ፡ ኵሉ፡ ገባሬ፡ ሰማያት፡ ወምድር፡ ዘያስተርኢ፡ ወዘኢያስተርኢ፡</incipit>
+     </msItem>
+
+</msItem>
+
+  <msItem xml:id="ms_i3">
+  <locus from="111r"/>
+  <title type="complete" xml:lang="en" ref="LIT1309Didesq">Didascalia Apostolorum</title>
+  <incipit xml:lang="gez">አንቀጽ፡ ፩ትምህርተ፡ ዲድስቅልያ፡ ዘአበው፡ ቅዱሳን፡ ሐዋርያት።</incipit>
+  <textLang mainLang="gez"></textLang>
+  </msItem>
+
+
+
+</msContents>
+<physDesc>
+  <objectDesc form="Codex">
+      <supportDesc>
+          <support>
+              <material key="parchment"/>
+          </support>
+          <extent>
+              <measure unit="leaf">145</measure>
+             <measure unit="leaf" type="blank">4</measure>
+              <dimensions type="outer" unit="in">
+                  <height>13.125</height>
+                  <width>10.625</width>
+              </dimensions>
+            </extent>
+            <foliation>Two unnumbered blank leaves at the beginning and two at the end.</foliation>
+      </supportDesc>
+      <layoutDesc>
+          <layout columns="3" writtenLines="38" cert="high">
+          </layout>
+      </layoutDesc>
+
+
+  </objectDesc>
+  <handDesc>
+    <handNote script="Ethiopic" xml:id="h1">
+          <desc>Written in a small, regulard hand, about the middle of the <date notBefore="1700" notAfter="1800">18th Century</date>.</desc>
+    </handNote>
+        </handDesc>
+
+        <additions>
+            <list>
+                <item xml:id="a1">
+                    <locus target="#1r"/>
+                    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+                        <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                    <q xml:lang="gez">መፍሐፈ፡ ኪዳን፡ ዘ<placeName ref="INS0101MadhaneAlam">መድኃኔ፡ ዓለም።</placeName></q>
+              </item></list>
+
+              <list>
+                <item xml:id="a2">
+                <locus target="#110r"/>
+                <desc type="Subscription">The note accompanying the <ref target="#ms_i2.23">Athanasian Creed</ref> states that it
+              was translated by <persName ref="PRS4709Giyorgis">Giyorgis of Saglā.</persName> </desc>
+                <q xml:lang="gez">ዘንተ፡ ሃይማኖተ፡ ጸሐፈ፡ አትናትዮስ፡ ዘሀገረ፡ ባባ፡ እንተ፡ ይእቲ፡ መንበረ፡ ጴጥሮስ፡ ወጳውሎስ፡ ወአምጽኦ፡ ምስር፡
+                ዛን፡ ዘብሔረ፡ አፍራቅያ፡ እስከ፡ ብሔረ፡ ኢትዮጵያ፡ ወአዕለውክዋ፡ አነ፡ ጊዮርጊስ፡ ሰግላዊ፡ እንዘ፡ እግዚእ፡ ይረድእ፡ ወቃሎ፡ ያጸንዕ፡ በትእምርት፡ ዘይተሉ፡ </q></item>
```

Comment:
This text does not  seem to be an addition, unless it was written by a secondary hand, which is not  the case I think. Thus it belongs to the corresponding msItem https://betamasaheft.eu/Guidelines/?q=rubric&id=manuscriptContents, see Example 7